### DISCUSSION TITLE
Ability to set default directories

### DISCUSSION THREAD

**Discussion by panda99a** [cdgriffith/FastFlix#476] [Ideas]
Thank you for a great program.

I like having the ability to load a whole directory for batch encoding, but I cannot find a way to change the Base Folder permanently.  It currently defaults to my User directory.  

I'm using the Load Directory feature mainly for converting TV series with multiple seasons, and having to navigate to another drive and nested subfolder for each set is a lot of clicks.

For this reason, I'd like to request any of the following:

1. A way to change the default Base Folder so I can point this where I'd like.
2. Having FastFlix remember the last folder I opened, or if it no longer exists, to move up a folder level.
3. The ability to recursively load subfolders of a folder that I select.  Thus if I have <TV series folder> with season 1, season 2, season 3 folders under it, if I open the <TV Series> folder, it would recursively load all the "season #" folders under it.

Of course, these are mainly convenience features, so not a huge priority, but are on the "nice to have" list.

Thanks!

**Comment by cdgriffith** [comment]
Hi @panda99a now in the Settings page under the File menu at the top, (or Ctrl+S) you can set a source directory! 

Make sure "No Default Source Folder" is unchecked. 

![Screenshot 2024-08-08 191845](https://github.com/user-attachments/assets/0b88bc25-1639-422f-b80c-f6601ec7bb15)

**Comment by dohn** [comment]
I subscribe to the ability to **recursively** load subfolders of a folder that I select.
